lbrynet/dht/protocol.py

```diff
@@ -118,7 +118,20 @@ def _migrate_outgoing_rpc_args(self, contact, method, *args):
             return args
         return args + ({b'protocolVersion': self._protocolVersion},)
 
+    @defer.inlineCallbacks
     def sendRPC(self, contact, method, args):
+        while True:
+            try:
+                response = yield self._sendRPC(contact, method, args)
+                return response
+            except TimeoutError:
+                if contact.contact_is_good:
+                    log.debug("RETRY %s ON %s", method, contact)
+                    continue
```

Comment:
I think there should be some delay here.

It seems like this could get into nasty feedback loops where some portion of sent rpc calls keep spinning while others get through with a high enough frequency to keep the contact marked as good.

Reply:
changed the infinite loop to obey the max rpcAttempts. So it will give up on high load + high packet loss. (sanity check)